modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+- New `fontsizeinput` toolbar button which allows the user to set the size via input and also increase and decrease it with `+` and `-` buttons. #TINY-9429
```

Comment:
it's not _really_ a button, and to be honest this applies to all the API names as well (the code is literally `const createFontSizeInputButton = (...) =>  createBespokeNumberInput(...)` which isn't creating a _button_)

```suggestion
- New `fontsizeinput` toolbar item which allows the user to set the size via input and also increase and decrease it with `+` and `-` buttons. #TINY-9429
```